.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '11'
```

Comment:
Is there an opportunity to set up a cache? https://github.com/actions/setup-java#caching-packages-dependencies

Reply:
Yup, I can add caching in.